lib/qesapdeployment.pm

```diff
@@ -444,7 +444,8 @@ sub qesap_execute {
 sub qesap_file_find_string {
     my (%args) = @_;
     foreach (qw(file search_string)) { croak "Missing mandatory $_ argument" unless $args{$_}; }
-    return script_run("grep \"$args{search_string}\" $args{file}");
+    my $ret = script_run("grep \"$args{search_string}\" $args{file}");
```

Comment:
code using this function is [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/0d6318282e6de791d4263d1cc7d1395ac1f9dcee/tests/sles4sap/qesapdeployment/deploy.pm#L27)